libs/ui/src/admin/NewsCard/NewsCardBase.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { FC } from 'react'
+
+import {
+  ButtonGroup,
+  HStack,
+  Image,
+  Stack,
+  Text,
+  useBreakpointValue,
+  VStack,
+} from '@chakra-ui/react'
+import { AiOutlineEye, AiOutlineLike, AiOutlineShareAlt } from 'react-icons/ai'
+import { BsBookmarkHeart } from 'react-icons/bs'
+
+import ActionButton from './ActionButton'
+import { NewsCardProps } from './types'
+
+export const NewsCardBase: FC<NewsCardProps> = ({
+  isFeatured,
+  news,
+  variant = 'horizontal',
+}) => {
+  const isVertical = useBreakpointValue({
+    base: true, // Always vertical in smaller viewports
+    lg: variant === 'vertical' ? true : false,
+  })
+  return (
+    <Stack
+      w={isVertical ? '300px' : '1000px'}
+      h={isVertical ? 'auto' : isFeatured ? '208px' : '150px'}
```

Comment:
```suggestion
      h={isVertical ? 'auto' : '200px'}
```
Card content will overflow if we have multiple lines of header and description with this height. Maybe we can limit description text lines on horizontal mode:

`<Text noOfLines={isVertical ? 3 : 2}>{description}</Text>`